Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:latest
 
+RUN apt-get update && apt-get install ca-certificates -y && apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
Possibly we can do this using this package like this: https://github.com/breml/rootcerts
```go
import (
    _ "github.com/breml/rootcerts"
)
```

But we can also do that in a separate PR